bin/plugin/commands/performance.js

```diff
@@ -251,99 +307,57 @@ async function runPerformanceTests( branches, options ) {
 			),
 			'utf8'
 		);
-
-		if ( options.wpVersion ) {
-			// In order to match the topology of ZIP files at wp.org, remap .0
-			// patch versions to major versions:
-			//
-			//     5.7   -> 5.7   (unchanged)
-			//     5.7.0 -> 5.7   (changed)
-			//     5.7.2 -> 5.7.2 (unchanged)
-			const zipVersion = options.wpVersion.replace(
-				/^(\d+\.\d+).0/,
-				'$1'
-			);
-			const zipUrl = `https://wordpress.org/wordpress-${ zipVersion }.zip`;
-			log( `        Using WordPress version ${ zipVersion }` );
-
-			// Patch the environment's .wp-env.json config to use the specified WP
-			// version:
-			//
-			//     {
-			//         "core": "https://wordpress.org/wordpress-$VERSION.zip",
-			//         ...
-			//     }
-			const confPath = `${ environmentDirectory }/.wp-env.json`;
-			const conf = { ...readJSONFile( confPath ), core: zipUrl };
-			await fs.writeFileSync(
-				confPath,
-				JSON.stringify( conf, null, 2 ),
-				'utf8'
-			);
-		}
-	}
-
-	// Printing the used folders.
-	log(
-		'\n>> Perf Tests Directory : ' +
-			formats.success( performanceTestDirectory )
-	);
-	for ( const branch of branches ) {
-		// @ts-ignore
-		const envPath = formats.success( branchDirectories[ branch ] );
-		log( `>> Environment Directory (${ branch }) : ${ envPath }` );
 	}
 
-	/*
-	 * 3- Running the tests.
-	 */
-
-	log( '\n>> Running the tests' );
+	logStep( 0, 'Looking for test files' );
 
 	const testSuites = getFilesFromDir(
-		path.join( performanceTestDirectory, 'test/performance/specs' )
-	).map( ( file ) => path.basename( file, '.spec.js' ) );
+		path.join( testRunnerDir, 'test/performance/specs' )
+	).map( ( file ) => {
+		logStep( 1, 'Found:', fAccent( file ) );
+		return path.basename( file, '.spec.js' );
+	} );
+
+	logStep( 0, 'Running tests' );
+
+	if ( wpZipUrl ) {
+		logStep( 1, 'Using:', fAccent( `WordPress v${ options.wpVersion }` ) );
+	} else {
+		logStep( 1, 'Using:', fAccent( 'WordPress trunk' ) );
+	}
 
-	const wpEnvPath = path.join(
-		performanceTestDirectory,
-		'node_modules/.bin/wp-env'
-	);
+	const wpEnvPath = path.join( testRunnerDir, 'node_modules/.bin/wp-env' );
 
 	for ( const testSuite of testSuites ) {
 		for ( let i = 1; i <= TEST_ROUNDS; i++ ) {
-			const roundInfo = `round ${ i } of ${ TEST_ROUNDS }`;
-			log( `    >> Suite: ${ testSuite } (${ roundInfo })` );
+			logStep(
+				1,
+				`Suite: ${ fAccent(
+					testSuite
+				) } (round ${ i } of ${ TEST_ROUNDS })`
+			);
```

Comment:
places like this could use some auto-formatting exemption. the previous string was easy to see at a glance, but the new one, split awkwardly over multiple lines, is not easy for someone like me to predict how it will look in the end.

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/54309/commits/52d04961ca96f2dd94dec02080a4cfc227a3e0a1